gpii/node_modules/lifecycleManager/src/LifecycleManager.js

```diff
@@ -36,6 +36,9 @@ var gpii = fluid.registerNamespace("gpii");
             nameResolver: {
                 type: "gpii.lifecycleManager.nameResolver"
             },
+            privateMatchMaker: {
```

Comment:
Please add a TODO note to explain that we will eventually factor this into the UserLogon request along where we make our other matchmaker requests